### PR TITLE
Add DIYRuZ_Geiger converter

### PR DESCRIPTION
**Pull request by nurikk**
Add converter

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -4186,6 +4186,16 @@ const converters = {
             return {action: `${button}_${action}`};
         },
     },
+    diyruz_geiger: {
+        cluster: 'msIlluminanceMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            return {
+                radioactive_events_per_minute: msg.data['61441'],
+                radiation_dose_perHour: msg.data['61442']
```
Comment (Koenkk) [inline converters/fromZigbee.js:4195]: `radiation_dose_perHour` -> `radiation_dose_per_hour`

File: converters/toZigbee.js [hunk 2]
```diff
@@ -2304,6 +2304,25 @@ const converters = {
             await entity.write('genOnOffSwitchCfg', payloads[key]);
         },
     },
+    diyruz_geiger_config: {
+        key: ['sensitivity', 'led_feedback', 'buzzer_feedback', 'sensors_count', 'sensors_type', 'alrert_treshold'],
```
Comment (Koenkk) [inline converters/toZigbee.js:2308]: `alrert_treshold` -> `alert_threshold`

File: converters/toZigbee.js [hunk 3]
```diff
@@ -2304,6 +2304,25 @@ const converters = {
...
+                'ON': 0x01,
+            };
+            const value = lookup.hasOwnProperty(rawValue) ? lookup[rawValue] : parseInt(rawValue, 10);
+            const payloads = {
+                sensitivity: {0xF000:  {value, type: 0x21}},
+                led_feedback: {0xF001: {value, type: 0x10}},
+                buzzer_feedback: {0xF002: {value, type: 0x10}},
+                sensors_count: {0xF003: {value, type: 0x20}},
+                sensors_type: {0xF004: {value, type: 0x30}},
+                alrert_treshold: {0xF005: {value, type: 0x23}},
```
Comment (Koenkk) [inline converters/toZigbee.js:2321]: `alrert_treshold` -> `alert_threshold`

**Comment by nurikk** [comment]
@Koenkk  Fixed

File: converters/toZigbee.js [hunk 4]
```diff
@@ -2298,6 +2298,25 @@ const converters = {
             await entity.write('genOnOffSwitchCfg', payloads[key]);
         },
     },
+    diyruz_geiger_config: {
+        key: ['sensitivity', 'led_feedback', 'buzzer_feedback', 'sensors_count', 'sensors_type', 'alert_treshold'],
```
Comment (Koenkk) [inline converters/toZigbee.js:2302]: `treshold` -> `threshold`

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**